app/serializers/admin/auction_serializer.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+  class AuctionSerializer < ActiveModel::Serializer
+    has_many :bids, seralizer: Admin::BidSerializer
+
+    attributes :issue_url,      :start_price,
```

Comment:
I find this format of listing args to be difficult to read and update. prefer:

```ruby
attributes(
  :issue_url,
 :start_price,
 :start_datetime,
)
```

that way it is also easy to alpha order to see if anything is missing!